tests/fixups/TraceFixup/DynamicLinkLibraryFixup.cpp

```diff
@@ -13,13 +13,39 @@
 auto AddDllDirectoryImpl = &::AddDllDirectory;
 DLL_DIRECTORY_COOKIE __stdcall AddDllDirectoryFixup(_In_ PCWSTR newDirectory)
 {
+	LARGE_INTEGER TickStart, TickEnd;
+	QueryPerformanceCounter(&TickStart);
     auto entry = LogFunctionEntry();
     auto result = AddDllDirectoryImpl(newDirectory);
+	QueryPerformanceCounter(&TickEnd);
 
     auto functionResult = from_win32_bool(result != 0);
     if (auto lock = acquire_output_lock(function_type::dynamic_link_library, functionResult))
     {
-        Log("AddDllDirectory:\n");
+		if (output_method == trace_method::eventlog)
+		{
+			std::string inputs = "";
+			std::string outputs = "";
+			std::string results = "";
+			
+			inputs = "Directory=" + InterpretStringA(newDirectory);
+
+			results = InterpretReturn(functionResult, result!=0).c_str();
+			if (function_failed(functionResult))
+			{
+				outputs +=  InterpretLastError();
+			}
+
+			std::ostringstream sout;
+			InterpretCallingModulePart1()
+				sout << InterpretCallingModulePart2()
+				InterpretCallingModulePart3()
+				std::string cm = sout.str();
+
+			Log_ETW_PostMsgOperationA("AddDllDirectory", inputs.c_str(), results.c_str(), outputs.c_str(),cm.c_str(), TickStart, TickEnd);
+		}
+		else
+		{        			Log("AddDllDirectory:\n");

```

Comment:
>        			 [](start = 3, length = 11)

formatting

Reply:
Addressed in commit 727a9e9856d361f686523f2ea2907876c3ef9b23 on 11/28